MiniApp/Classes/Display/MiniAppWebView.swift

```diff
@@ -10,6 +10,8 @@ internal class MiniAppWebView: WKWebView {
         config.preferences.javaScriptEnabled = true
         config.preferences.setValue(true, forKey: "allowFileAccessFromFileURLs")
         config.setURLSchemeHandler(URLSchemeHandler(versionId: versionId), forURLScheme: schemeName)
+        config.allowsInlineMediaPlayback = true
+        config.allowsPictureInPictureMediaPlayback = true
```

Comment:
Does the mini app need to add anything to use picture-in-picture? Or just adding `playsinline` will enable both?

Reply:
No, this is the default behavior for all videos into a basic web view. In fact it seems like adding `playsinline` disables PiP. I just added this to the config to consolidate the default value, so we have a consistent behavior in Mini App SDK if the default value of this parameter was ever changed by Apple in a future update.